tool/src/org/antlr/v4/codegen/target/GoTarget.java

```diff
@@ -71,15 +73,20 @@ protected void addBadWords() {
 	@Override
 	protected void genFile(Grammar g, ST outputFileST, String fileName) {
 		super.genFile(g, outputFileST, fileName);
-		gofmt(fileName); // consider commenting out when debugging
+		if (System.getenv("ANTLR_GO_DISABLE_GOFMT") == null
```

Comment:
For boolean flags, allowing an empty string to mean the same as it not being defined at all can help aid scripting and other automated invocations because you don't have to deal with string concatenation logic. E.g. `ANTLR_GO_DISABLE_GOFMT=$USEGOFMT antlr4 ...`.

Same for the Java system property flag. Allowing `-Dantlr-go-disable-gofmt=1` (or `=0`) in addition to `-Dantlr-go-disable-gofmt` enables things like `antlr4 -Dantlr-go-disable-gofmt=$USEGOFMT ...`.